src/miral/runner.cpp

```diff
@@ -142,6 +143,17 @@ try
 
         // Provide the command line and run the server
         server->set_command_line(argc, argv);
+
+        // TODO: This is a band-aid solution, and perhaps not the best one. The mir core libraries provide a default
+        // window manager, but mirAL would like to provide a different default window manager.
+        server->override_the_default_window_manager_builder([&server](mir::shell::FocusController* focus_controller) -> std::shared_ptr<mir::shell::WindowManager>
```

Comment:
Can't we just call `override_the_window_manager_builder()` before `for (auto& option : options)` and drop the new function?

Reply:
Yes!